bookkeeper-server/src/main/java/org/apache/bookkeeper/bookie/Bookie.java

```diff
@@ -857,7 +884,13 @@ public synchronized void start() {
 
         ledgerStorage.start();
 
-        syncThread.start();
+        // check the bookie status to start with
+        if (forceReadOnly.get()) {
```

Comment:
if force read only and read status from disk are mutually exclusive then this needs to be documented somewhere (that is, which one has precedence), right?

Reply:
So force is only one-way operation; we have no forceWrite option. i.e if bookieStatus says readonly there is no way to start bookie in RW until we change bookieStatus. I am still at large don't understand the need of this change, but as long as we document it well, I am fine. I think it is mostly an operational overhead than letting operational scripts deal with starting bookie in R/RW.